.circleci/config.yml

```diff
@@ -36,14 +41,71 @@ workflows:
 
 jobs:
 
+  test_39: &test-template
+    <<: *defaults
+
+    steps:
+      - checkout
+
+      - run:
+          name: Test minimal openapi-core version on Python 3.9
+          command: ./inject_minimal_openapi-core.sh
```

Comment:
This file probably needs to be modified to add support for another Python version.